main.rb

```diff
@@ -0,0 +1,41 @@
+require_relative './app'
+
+def menu
+  puts 'Welcome to School Library app'
+  puts 'Select an option by entering a number: '
+  puts '1) List all books'
+  puts '2) List all people'
+  puts '3) Create a person'
+  puts '4) Create a book'
+  puts '5) Create a rental'
+  puts '6) List all rentals for a given person id'
+  puts '7) Exit'
+end
+
+def main
+  print 'welcome!'
+  app = App.new
+  loop do
+    puts menu
+    option = gets.chomp.to_i
+    options(option, app)
+  end
+end
+
+def options(option, app)
+  case option
+  when 1
+    app.list_of_all_books
+  when 2
+    app.list_of_all_people
+  when 3
+    create_person(app)
```

Comment:
- [ ] Please fix this error with the create_person method, it seems that you didn't create the method at all.
![image](https://user-images.githubusercontent.com/74744787/211101417-b89fdd20-7db5-4009-b56e-08ce5e492e4f.png)

Reply:
Oh, I see I missed it. thanks for your great comment. I will fix it right now.